docs/reference/administration_reference.md

```diff
@@ -0,0 +1,96 @@
+# Administration reference guide
+
+## Overview
+
+This document provides a reference guide for all the attributes available for `invenio-administration` views.
+
+## List View
+
+### Attributes
+| Attribute                     | Required   | Default                                | Description                                                                                                          |
+|-------------------------------|------------|----------------------------------------|----------------------------------------------------------------------------------------------------------------------|
+| **name**                      |            | *View class name*                      | - view's name used by Flask to create routing                                                                        |
+| **resource_config**           | *required* |                                        | used to retrieve the resource (refers to extension attribute - check ext.py)                                         |
```

Comment:
I am not sure that `refers to extension attribute - check ext.py` is super clear